docs/standards/smirnoff.md

```diff
@@ -359,9 +359,27 @@ Later revisions will also provide support for special interactions using the `<A
 </vdW>   
 ```
 
+Version 0.4 includes a `long_range_treatment` attribute (default: `"isotropic"`) that controls how long-range vdW iteractions are handled for periodic systems; this attribute is ignored for non-periodic systems.
+
+Supported values of `long_range_treatment` are
+  * `"none"`: No long-range vdW correction is applied.
+  * `"isotropic"`: An isotropic vdW correction, described below, is applied.
+  * `"LJPME"`: An Ewald sum is used (commonly referred to as LJPME) to treat the vdW potential in a periodic manner [1](https://doi.org/10.1063/1.464397) [2](http://dx.doi.org/10.1063/1.465608) [3](https://doi.org/10.1021/acs.jctc.5b00726). Note that this is only compatible with certain forms of the potential that involve sums of inverse even powers of `r`.
+
+For the `"isotropic"` case, the vdW correction is computed as
```

Comment:
For "isotropic" one needed clarification is that this is explicitly a **potential** switch, which is not necessarily the default for many MD engines (for example, for gromacs "force-switch" is the default.  Force switch is removes a weird discontinuity in the switching region, at the cost of altering the potential at all r (thus leaving the force the same). In many cases, however, the force-switch results in being lambda-dependent, with a different switch depending on the lambda, so give incorrect results - I don't recall if this has been addressed in GROMACS yet).